src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
You mean because the *first* level is skipped in effects coding? Indeed, that's kind of unusual; are there examples of other software doing this?

BTW, this would be nicely solved by printing the (pseudo-)coefficient for the skipped level, which users are likely to compute manually anyway. Do you have opinions on point 2 of https://github.com/JuliaStats/GLM.jl/issues/29?